.eleventy.js

```diff
@@ -44,6 +44,23 @@ const md = markdownIt(markdownItConfig)
   .use(markdownItContainer, 'note')
   .use(markdownItContainer, 'table-wrapper')
   .use(markdownItContainer, 'ecmascript-algorithm')
+  .use(markdownItContainer, "figure",  {
```

Comment:
Figure seems like a confusing name, given that we have special `figure` tag. Maybe `inline-figure` or `inline-svg`?

Reply:
I added this for explicit figures (instead of implicit figures with a single-line image), so as discussed offline it's fine as-is. Using actual html tags would have failed the "img in figure" check in our image embedding script.